<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paysages 3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../data/ui_pictures.qrc">
    <normaloff>:/logo/images/logo_32.png</normaloff>:/logo/images/logo_32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="tool_panel" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Paysages 3D&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;img src=&quot;:/logo/images/logo_32.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>Preview</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>3D</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>Top</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>3</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Actions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="button_explore">
            <property name="text">
             <string>Explore in 3D</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../data/ui_pictures.qrc">
              <normaloff>:/buttons/logo/images/explore.png</normaloff>:/buttons/logo/images/explore.png</iconset>
            </property>
            <property name="shortcut">
             <string>F2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_render">
            <property name="text">
             <string>Render</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../data/ui_pictures.qrc">
              <normaloff>:/buttons/logo/images/render.png</normaloff>:/buttons/logo/images/render.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>File</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="button_file_new">
            <property name="text">
             <string>New</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../data/ui_pictures.qrc">
              <normaloff>:/buttons/logo/images/new.png</normaloff>:/buttons/logo/images/new.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_file_save">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../data/ui_pictures.qrc">
              <normaloff>:/buttons/logo/images/save.png</normaloff>:/buttons/logo/images/save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_file_load">
            <property name="text">
             <string>Load</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../data/ui_pictures.qrc">
              <normaloff>:/buttons/logo/images/load.png</normaloff>:/buttons/logo/images/load.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <widget class="MainTerrainForm" name="tab_terrain">
       <attribute name="icon">
        <iconset resource="../../../../data/ui_pictures.qrc">
         <normaloff>:/tabs/images/tab_terrain.png</normaloff>:/tabs/images/tab_terrain.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Lanscape shape</string>
       </attribute>
      </widget>
      <widget class="MainTexturesForm" name="tab">
       <attribute name="icon">
        <iconset resource="../../../../data/ui_pictures.qrc">
         <normaloff>:/tabs/images/tab_textures.png</normaloff>:/tabs/images/tab_textures.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Textures</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_file_new"/>
   <addaction name="action_file_save"/>
   <addaction name="action_file_load"/>
   <addaction name="separator"/>
   <addaction name="action_explore"/>
   <addaction name="action_quick_render"/>
   <addaction name="action_final_render"/>
   <addaction name="separator"/>
   <addaction name="action_about"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>946</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_file_new"/>
    <addaction name="action_file_save"/>
    <addaction name="action_file_load"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="action_explore"/>
    <addaction name="action_quick_render"/>
    <addaction name="action_final_render"/>
    <addaction name="action_last_render"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_explore">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/explore.png</normaloff>:/buttons/logo/images/explore.png</iconset>
   </property>
   <property name="text">
    <string>Explore in 3D</string>
   </property>
   <property name="toolTip">
    <string>Start exploring your scenery in real-time 3D</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="action_file_new">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/new.png</normaloff>:/buttons/logo/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New scene</string>
   </property>
   <property name="toolTip">
    <string>Create a virgin scene</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_file_load">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/load.png</normaloff>:/buttons/logo/images/load.png</iconset>
   </property>
   <property name="text">
    <string>Load scene</string>
   </property>
   <property name="toolTip">
    <string>Load a scenery from a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_file_save">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/save.png</normaloff>:/buttons/logo/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save scene</string>
   </property>
   <property name="toolTip">
    <string>Save a scenery to a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_quick_render">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/render.png</normaloff>:/buttons/logo/images/render.png</iconset>
   </property>
   <property name="text">
    <string>Quick render</string>
   </property>
   <property name="toolTip">
    <string>Start a quick render</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_final_render">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/render.png</normaloff>:/buttons/logo/images/render.png</iconset>
   </property>
   <property name="text">
    <string>Final render</string>
   </property>
   <property name="toolTip">
    <string>Start the final rendering processs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/about.png</normaloff>:/buttons/logo/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Display information on this software</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_last_render">
   <property name="icon">
    <iconset resource="../../../../data/ui_pictures.qrc">
     <normaloff>:/buttons/logo/images/render.png</normaloff>:/buttons/logo/images/render.png</iconset>
   </property>
   <property name="text">
    <string>Show last render</string>
   </property>
   <property name="toolTip">
    <string>Display the last render done</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainTerrainForm</class>
   <extends>QWidget</extends>
   <header>terrain/mainterrainform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainTexturesForm</class>
   <extends>QWidget</extends>
   <header>textures/maintexturesform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../data/ui_pictures.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_explore</sender>
   <signal>clicked()</signal>
   <receiver>action_explore</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_render</sender>
   <signal>clicked()</signal>
   <receiver>action_final_render</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_file_new</sender>
   <signal>clicked()</signal>
   <receiver>action_file_new</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_file_load</sender>
   <signal>clicked()</signal>
   <receiver>action_file_load</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_file_save</sender>
   <signal>clicked()</signal>
   <receiver>action_file_save</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
